### PR TITLE
DOCSP-27027 unique and nonUnique indexes

### PR DESCRIPTION
**Pull request by davemungo**
[STAGING](https://docs-mongodbcom-staging.corp.mongodb.com/cluster-sync/docsworker-xlarge/DOCSP-27027-unique-nonUnique-limitation-v1.0/reference/limitations/)

[JIRA](https://jira.mongodb.org/browse/DOCSP-27027)

### CODE REVIEW

**Review by jmd-mongo — APPROVED** [review]
Thanks for this update, @davemungo. This one LGTM!

Thanks,
Joe

**Comment by davemungo** [comment]
@edobranov would you review this docs update when you have a moment? Thanks

**Review by edobranov — APPROVED** [review]
LGTM!

File: source/reference/limitations.txt [hunk 1]
```diff
@@ -62,6 +62,8 @@ General Limitations
   downgrade, you should drop all non-system databases in the destination
   cluster before starting the new binary. Syncing operations will
   restart from the beginning when you start the new processes.
+- You cannot sync a collection that has a unique index and a non-unique
+  index defined on the same field(s).
```
Comment (jeff-allen-mongo) [inline source/reference/limitations.txt:66]: I'm not blocking this PR on this comment, but I just wanted to mention that our style guide says:

> Don't add (s) or (es) to the end of a noun to indicate the possibility of more than one item. Use the singular form or the plural form, or use both forms joined by a conjunction.

See [Punctuation](https://www.mongodb.com/docs/meta/style-guide/style/punctuation/#parentheses).

Still going to merge this one as-is.

**Pull request merged**